src/System.Diagnostics.Process/tests/ProcessWaitingTests.cs

```diff
@@ -141,7 +143,7 @@ public void SingleProcess_CopiesShareExitInformation()
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/corefx/issues/22269", TargetFrameworkMonikers.UapNotUapAot)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapNotUapAot, "Getting handle of child process running on UAP is not possible")]
```

Comment:
I would prefer not disabling the test for uapaot from now. to ensure at least no regressions when running outside app containers (.NetNative Linux scenario)